external/dhcpd/dhcpd.c

```diff
@@ -1637,7 +1639,7 @@ int dhcpd_run(void *arg)
 
 exit_with_error:
 	g_dhcpd_running = 0;
-
+    g_dhcp_sta_joined = 0;
```

Comment:
If callback initialization is handled in dhcpd_start function, you don't have to set it to NULL in run function. In start initialize by default to NULL, if the parameter is valid then it gets assigned below  or else it remains NULL.

Reply:
Thanks, I will fix it